tests/test_claim_state_duty.py

```
@@ -0,0 +1,228 @@
+        requests=["Взыскать задолженность 1 000 000 тенге."],
+    )
+    context = "Истец: Иванов Иван, ИИН 900101300001. Ответчик спорит, что истец является потребителем."
+    decision = decide_state_duty(context, _research(), draft)
+    assert decision.mode == "property"
+    assert decision.amount == 10_000
+    assert decision.deferred is False
+
+
+def test_disability_exemption_is_not_overwritten_by_property_rate():
+    draft = _draft(
+        claimant=["Иванов Иван, лицо с инвалидностью, ИИН 900101300001, адрес: г. Алматы"],
```

Ruff (0.16.1)

[warning] String contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF001)